bucket/opencv.json

```
@@ -0,0 +1,32 @@
+{
+    "version": "4.5.5",
+    "description": "Open Source Computer Vision Library",
+    "homepage": "https://opencv.org/",
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/opencv/opencv/releases/download/4.5.5/opencv-4.5.5-vc14_vc15.exe#/dl.7z",
+            "hash": "cac31973cd1c59bfe9dc926acbde815553d23662ea355e0414b5e50d8f8aa5a8"
+        }
+    },
+    "extract_dir": "opencv",
+    "env_set": {
+        "OPENCV_DIR": "$dir\\build\\x64\\vc15"
+    },
+    "env_add_path": "$dir\\build\\x64\\vc15\\bin",
```

```suggestion
    "env_add_path": "x64\\vc14\\bin",
```

The included .bat file uses vc14 folder, so we should also use that.